nests.json

```diff
@@ -60,7 +60,7 @@
       "packages": [
         "SurfaceCtrl"
       ],
-      "description": "A 3D viewer few different 3D object data format",
+      "description": "An OpenGL 3D viewer supporting few differents 3D file format",
```

Comment:
@Xemuth ,

Just a suggestion: 

"An OpenGL 3D object viewer, supporting various 3D file formats"

Reply:
@ismail-yilmaz yes, it's better